svelte/components/Global/Home.svelte

```diff
@@ -63,9 +70,18 @@
   <span slot="content">
     {#if chainId && account && address}
       {#if tokenID}
-        <Nft {chainId} {address} {tokenID} />
+        <Nft {chainId} {address} bind:tokenID {account} {platform} />
       {:else}
-        <NftsList {chainId} {address} {account} {refresh} bind:refreshing />
+        <NftList
+          {chainId}
+          {address}
+          bind:tokenID
+          {account}
+          {refresh}
+          bind:mainContentDisplayComponent
```

Comment:
to move inside NftList component

Reply:
talking about bind:mainContentDisplayComponent?